### PR TITLE
Add more parameters and functions to BinaryTrajectories

### PR DESCRIPTION
**Pull request by knelli2**
These things will be helpful for testing control systems:
- `velocity` parameter. The CoM of the binary will move with this
  constant velocity. Default is zero velocity.
- `newtonian` bool to indicate using Newtonian trajectories instead
  of PN trajectories (Default is false, i.e. PN trajectory)
- `angulary_velocity` function. Returns the angular velocity.
- `positions_no_expansion` function. Regardless of Newtonian or PN,
  the orbit is circular, but the frequency of the orbit follows
  either Newtonain or PN, whetever was specified. (Note: these
  positions are not physical, just useful for testing control
  systems)

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Comment by nilsdeppe** [comment]
Can you specify what you're adding to `BinaryTrajectories` in the title? My view is if a PR does so much stuff it can't be succinctly described in the title it might be doing too much (not in this case, I think). I also don't know what `Newtonian` means in the PR description, maybe expand on that?

**Comment by knelli2** [comment]
Updated the title and description in both the PR and commit.

File: tests/Unit/Helpers/PointwiseFunctions/PostNewtonian/BinaryTrajectories.cpp [hunk 1]
```diff
@@ -9,22 +9,63 @@
...
+  xA += velocity_[0] * time;
+  yB += velocity_[1] * time;
+  yA += velocity_[1] * time;
+  const double zB = velocity_[2] * time;
+  const double zA = zB;
+  return {{xA, yA, zA}, {xB, yB, zB}};
+}
+
+std::pair<std::array<double, 3>, std::array<double, 3>>
+BinaryTrajectories::positions_no_expansion(const double time) const {
```
Comment (wthrowe) [inline tests/Unit/Helpers/PointwiseFunctions/PostNewtonian/BinaryTrajectories.cpp:44]: This differs by one line from the previous function.  Factor the common code into a helper.

**Review by wthrowe — CHANGES_REQUESTED** [review]

**Comment by knelli2** [comment]
Squashed changes directly because they were short.

**Review by wthrowe — APPROVED** [review]

**Review by geoffrey4444 — APPROVED** [review]

**Pull request merged**